CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # (Unreleased; add upcoming change notes here)
+# 0.4.0
+- iodide notebooks now automatically save to the server
+- removed `ctrl+s` as a keybinding
```

Comment:
I think you can skip the last bullet. Also, it doesn't matter so much, but I don't think it's necessary to pre-emptively fill in the next version number. Whoever bumps the changelog can assign their own.

Reply:
np, wasn't really thinking deeply about that :)